ambari/README.md

```diff
@@ -0,0 +1,46 @@
+# Agent Check: Ambari
+
+## Overview
+
+This check monitors [Ambari][1] through the Datadog Agent.
+
+## Setup
+
+### Installation
+
+The Ambari check is included in the [Datadog Agent][2] package.
+No additional installation is needed on your server.
+
+### Configuration
+
+1. Edit the `ambari.d/conf.yaml` file, in the `conf.d/` folder at the root of your Agent's configuration directory to start collecting your ambari performance data. See the [sample ambari.d/conf.yaml][2] for all available configuration options.
+
+2. [Restart the Agent][3].
+
+### Validation
+
+[Run the Agent's status subcommand][4] and look for `ambari` under the Checks section.
+
+## Data Collected
+
+### Metrics
+
+Ambari does not include any metrics.
```

Comment:
If there are no metrics, service checks, or events, what does this integration do?

Reply:
hahaha, this is the template for the README. I have not filled it in yet